docs/source/guide/build_pipelines/pipeline_basics.rst

```diff
@@ -55,7 +61,7 @@ where ``<pipeline_name>`` is ``iris_pipeline`` in the current example.
 .. note::
 
    If you want hands-on practice,
-   check out `this <https://github.com/LineaLabs/lineapy/blob/main/examples/tutorials/pipeline_basics.ipynb>`_ tutorial notebook.
+   check out `this <https://github.com/LineaLabs/lineapy/blob/main/examples/tutorials/02_build_pipelines.ipynb>`_ tutorial notebook.
```

Comment:
we need to figure out a reliable way to update the links here. it always points to main right now but ideally we'll want it to take us to the branch that the docs were built off of. This can be done using external reference plugins and using xref but there might be a better way i feel. See xref example [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html)

Reply:
Shardul and I discussed this and decided that we can circle back to this issue after the launch.